code/parachain/frame/pablo/src/lib.rs

```diff
@@ -828,11 +832,13 @@ pub mod pallet {
 				T::Assets::transfer(fees.asset_id, who, owner, fees.owner_fee, false)?;
 			}
 			if !fees.protocol_fee.is_zero() {
-				T::ProtocolStaking::transfer_reward(
+				T::ProtocolStaking::transfer_earnings(
 					who,
 					&T::PbloAssetId::get(),
 					fees.asset_id,
 					fees.protocol_fee,
+					// REVIEW(benluelo): Should this be kept alive?
```

Comment:
We shouldn't generally as the account is known in advance and more we keep alive the more storage the chain uses

Reply:
I'm not sure -`disburse_fees` is called in `exchange`, which in turn called in `buy` which is called with `ensure_signed(origin)?` as `who`, and the `buy` extrinsic (and `Amm::buy`) takes a `keep_alive` parameter which is passed to `exchange` but not `disburse_fees`